deploy/terraform/aws-ec2/install_frontend_host.sh.tpl

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+cat << EOF >> /home/bitnami/.profile
+export DD_SERVICE=store-frontend
+export DD_LOGS_INJECTION=true
+export DD_TRACE_ANALYTICS_ENABLED=true
+export DB_USERNAME=datadog
+export DB_PASSWORD=password
+export DD_CLIENT_TOKEN=${clienttoken}
+export DD_APPLICATION_ID=${appid}
+EOF
+
+source  /home/bitnami/.profile
+
+# gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 \
+# 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+
+
+wall -n "installing"
+export DEBIAN_FRONTEND=noninteractive && \
+    sudo apt-get update && \
+    sudo apt-get install -y --no-install-recommends curl build-essential git-core libsqlite3-dev libpq-dev ruby && \
+    curl -sL https://deb.nodesource.com/setup_12.x | sudo bash - && \
+    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add - && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
+    sudo apt-get update && \
+    sudo apt-get install -y nodejs yarn && \
+
+# curl -sSL https://get.rvm.io | bash -s stable --ruby
```

Comment:
Is this for future use or was it while you tested out stuff?

Reply:
i did that because it was in the source containers

Reply:
wait, is the comment about line 29 or 27

Reply:
if 29, then fixed